pulp_python/app/serializers.py

```diff
@@ -21,6 +21,31 @@ class Meta:
         model = python_models.Classifier
         fields = ('name',)
 
+class ProjectSpecifierSerializer(serializers.ModelSerializer):
+    """
+    A serializer for Python Classifiers
+    """
+
+    name = serializers.CharField(
+        help_text=_("A python project name.")
+    )
+
+    version = serializers.CharField(
+        help_text=_("versionspecifier"),
+        required = False,
+        allow_blank=True
+    )
+
+    sha256 = serializers.CharField(
+        help_text=_("hash"),
+        required=False,
+    )
+
```

Comment:
If we add a `create` function, we could accept strings in the `requirements.txt` syntax instead a dictionary.